apps/sms/services/README.md

```diff
@@ -0,0 +1,91 @@
+# The layers in the Messages application
+
+## Terms explanation
+
+There are 3 layers whose interfaces managed by the Bridge (aka [threads.js](https://github.com/gaia-components/threads/)):
+
+* Services define high-level semantics to access the model. They are the only
```

Comment:
> I see, so the last thing to clarify here, do you want to see level of abstraction around bridge.client (like I'm trying to do with ActivityClient) or you think we should use bridge.client directly from view (like direct call in JS :))?

I'm a bit concerned to have as many "client" that we have services. Could be difficult (or at least cumbersome) to keep in sync properly. That's my only issue with this.

Also it depends if we want to think "object" (with "remote" methods -- also called RPC for remote procedure calls) or "events".

We can start with a client and we'll see later if it's painful.

> Yep I meant this, will keep it in my head as "service-shim_for_api_that_is_not_available_in_worker" then :) For the name, I'm fine with Wrapper or Shim.

I thought Wrapper is the terminology used in https://wiki.mozilla.org/Gaia/Architecture_Proposal but it's not. So we can call it Shim if you prefer.